.github/workflows/ci.yml

```diff
@@ -14,6 +14,9 @@ concurrency:
 jobs:
   ci:
     runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 1
```

Comment:
I also enabled Docker buildkit as the default builder here. It was enabled everywhere else, so I guessed that it missing here was an oversight.